sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/CollectionConfiguration.cs

```diff
@@ -121,6 +142,92 @@ private void UpdateMetricIdOfError(CollectionConfigurationError error, string id
             }
         }
 
+        private void CreateDocumentStreams(
+            out CollectionConfigurationError[] errors,
+            IEnumerable<DocumentStream> previousDocumentStreams)
+        {
+            var errorList = new List<CollectionConfigurationError>();
+            var documentStreamIds = new HashSet<string>();
+
+            // quota might be changing concurrently on the collection thread, but we don't need the exact value at any given time
+            // we will try to carry over the last known values to this new configuration
+            Dictionary<string, Tuple<float, float, float, float, float>> previousQuotasByStreamId =
+                previousDocumentStreams.ToDictionary(
+                    documentStream => documentStream.Id,
```

Comment:
We don't have documentStream, should this change?

Reply:
We do have documentStream. This PR adds it. What is discarded is the `documentStreamIds` field in `DocumentIngress` types such as `Request` and `RemoteDependency`.